persistent/Database/Persist/Compatible.hs

```diff
@@ -0,0 +1,9 @@
+module Database.Persist.Compatible
+  ( Compatible(..)
+  , makeCompatibleInstances
+  , makeCompatibleKeyInstances
+  ) where
```

Comment:
Can we stick with 4 space indentation here? I know the codebase isn't consistent, but I want to get it moving in that direction. Thanks!

Reply:
Sure! Is it just the first indent that's 4-space, and should I convert the other files to it as well?

Reply:
Don't worry about converting anything old, but anything new should be 4 space. Thanks!

Reply:
Alright, I hope that's done correctly.